pkg/users/account-details.js

```diff
@@ -145,13 +168,15 @@
                                     console.log("Account locked state doesn't match desired value, trying again.");
                                     // only retry once to avoid uncontrolled recursion
                                     change_locked(value, true);
-                                } else
-                                    set_edited_locked(null);
+                                }
                             });
                 })
                 .catch(error => {
-                    set_edited_locked(null);
                     show_unexpected_error(error);
+                    setIsLocked(!value);
```

Comment:
This added line is not executed by any test.  [Details](https://logs-cockpit.apps.ocp.cloud.ci.centos.org/pull-21865-c0f5f5da-20250814-145542-fedora-42-devel/Coverage/lcov/github-pr.diff.gcov.html)